name-registry/tests/functions/expiry.rs

```diff
@@ -0,0 +1,47 @@
+mod success {
+    use crate::utils::{
+        abi::{expiry, extend, register},
+        setup, EXTEND_DURATION, REGISTER_DURATION,
+    };
+    use fuels::prelude::*;
+
+    #[tokio::test]
+    async fn can_get_expiry() {
+        let (instance, _id, wallet, _wallet2) = setup().await;
+        let wallet_identity = Identity::Address(Address::from(wallet.address()));
+        let name = String::from("SwaySway");
```

Comment:
These seem to be getting reused here and there. Perhaps a default struct should be returned so that you can set it up once it the `setup()` and then use the values as you see fit. If you need an exceptional value then that can be defined in the test itself